packages/contract-wrappers/CHANGELOG.json

```diff
@@ -1,10 +1,14 @@
 [
     {
-        "version": "1.1.0-rc.2",
+        "version": "1.0.1-rc.3",
```

Comment:
@fabioberger not sure if I'm doing this rc version bumping correctly. If we're on an rc package do we just bump the rc version for any type change and keep the left side the same?

Reply:
I accidentally bumped it to 1.1.0-rc.2 in a previous PR but since this hasn't been published yet its safe to bump down

Reply:
Double checked with fabio offline, we're supposed to just bump the right side for any change while it is RC